contracts/PriceOracle/PriceOracle.sol

```diff
@@ -201,10 +192,24 @@ contract PriceOracle is Ownable2Step {
      */
     function validateTokenConfig(TokenConfig memory config) internal view {
         if (config.cToken == address(0)) revert MissingCTokenAddress();
-        // Dual check of field being present and being non-zero
-        if (config.baseUnit == 0) revert InvalidBaseUnit();
```

Comment:
Why was this check removed?

Reply:
Since we replaced the `baseUnit` config with `underlyingAssetDecimals`, we don't need this check anymore. The validation that replaced this is in `validateDecimals` now.